week-3/Banking/Banking.sol

```diff
@@ -0,0 +1,91 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.8.17;
+
+contract Banking {
+    struct Account {
+        address payable walletAddress;
+        uint256 balance;
+    }
+
+    // Banks customer database
+    mapping(address => Account) internal recordBook;
```

Comment:
If you are using mapping, you can simply ignore Account struct. ```mapping(address => uint256) internal addressToBalance; ``` would be enough as the Struct doesn't contain any other info in it.